html/modules/custom/reliefweb_post_api/schemas/v2/training.json

```diff
@@ -54,7 +54,7 @@
             "maxItems": 2
         },
         "country": {
-            "description": "Training country(ies) as a list of IDs from https://api.reliefweb.int/v1/countries. Mandatory when the training format contains on-site otherwise not allowed.",
+            "description": "Training country(ies) as a list of IDs from https://api.reliefweb.int/v1/countries. Mandatory when the training format is 'on-site', otherwise not allowed.",
```

Comment:
"is 'on-site'" is not correct because it is possible for a training to be both on-site and online thus the original "contains".

Reply:
Good point - adjusted.